packages/visualizations/src/components/Table/Cell/Cell.svelte

```diff
@@ -1,46 +1,66 @@
 <script lang="ts">
     import type { Column } from '../types';
     import Format, { isValidRawValue } from './Format';
+    import { stickyColumnsOffset, lastStickyColumn } from '../store';
 
     export let rawValue: unknown;
     export let column: Column;
+    export let isHorizontallyScrolled: boolean;
 
     $: ({ dataFormat, options = {} } = column);
 </script>
 
 <!-- To display a format value, rawValue must be different from undefined or null -->
-<td class={`table-data--${dataFormat}`}>
-    {#if isValidRawValue(rawValue)}
-        <svelte:component this={Format[dataFormat]} {rawValue} {...options} />
-    {/if}
+<td
+    style={`--sticky-offset: ${$stickyColumnsOffset.get(column.key)}px;`}
+    class:sticky={$stickyColumnsOffset.has(column.key)}
+    class:isLastSticky={column.key === $lastStickyColumn}
+    class:isHorizontallyScrolled
+>
+    <div class={`table-data--${dataFormat}`}>
+        {#if isValidRawValue(rawValue)}
+            <svelte:component this={Format[dataFormat]} {rawValue} {...options} />
+        {/if}
+    </div>
 </td>
 
-<style>
+<style lang="scss">
+    @import '../sticky';
     :global(.ods-dataviz--default td) {
-        padding: var(--spacing-75);
+        background-color: white;
```

Comment:
<img width="1713" alt="image" src="https://github.com/user-attachments/assets/c87d3a34-a7dd-4b6a-bd4e-f7ec796be122">

We lost the ability to apply a background color on tbody, tr

Maybe we could move this style to the tr tag?

Reply:
It has to be on the td because we will have overlapping tds with no background elese. I changed the customization stories, it seems acceptable.